tensorboard/webapp/feature_flag/actions/feature_flag_actions.ts

```diff
@@ -43,3 +43,16 @@ export const featureFlagOverrideChanged = createAction(
     flags: Partial<FeatureFlags>;
   }>()
 );
+
+export const featureFlagOverridesReset = createAction(
+  '[FEATURE FLAG] Resetting feature flag overrides',
+  props<{
+    flags: Array<keyof FeatureFlags>;
+  }>()
+);
+
+export const featureFlagsOverridesAllSetToDefault = createAction(
+  '[FEATURE FLAG] Resetting all feature flag overrides',
+  // NgRx does not allow actions without props.
```

Comment:
I don't think that's true.

Here are some actions without props:
https://cs.opensource.google/tensorflow/tensorboard/+/master:tensorboard/webapp/core/actions/core_actions.ts;l=39;drc=367839f30de3968eff5fecb8b38e1d5782434b6b

Reply:
🤯 that seems really obvious seeing it now but I couldn't figure it out.